packages/lbfgs/lbfgs.0.9.3/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/L-BFGS-ocaml"
+dev-repo: "git+https://github.com/Chris00/L-BFGS-ocaml.git"
+bug-reports: "https://github.com/Chris00/L-BFGS-ocaml/issues"
+doc: "https://Chris00.github.io/L-BFGS-ocaml/doc"
+tags: [ "science" "numerics" "optimization" "minimization" "maximization"
+        "bound-constrained" "large-scale" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc"] {with-doc}
+]
+depends: [
+  "ocaml"
+  "base-bigarray"
+  "base-bytes"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
According to your `dune-project` file this requires at least dune 1.1

Reply:
Right — thanks!